src/plugins/data_views/server/rest_api_routes/public/runtime_fields/get_runtime_field.ts

```diff
@@ -44,15 +44,22 @@ export const getRuntimeField = async ({
   name,
 }: GetRuntimeFieldArgs) => {
   usageCollection?.incrementCounter({ counterName });
-  const dataView = await dataViewsService.get(id);
+  const dataView = await dataViewsService.getDataViewLazy(id);
 
   const field = dataView.getRuntimeField(name);
 
   if (!field) {
     throw new ErrorIndexPatternFieldNotFound(id, name);
   }
 
-  return { dataView, fields: Object.values(dataView.getFieldsByRuntimeFieldName(name) || {}) };
+  return {
+    dataView,
+    fields: Object.values(
+      (
+        await dataView.getFields({ fieldName: [name], mapped: false, scripted: false })
+      ).getFieldMap()
```

Comment:
Does `getFields` with `mapped: false, scripted: false` return only runtime fields? Not sure exactly if that's what this is supposed to do.

Reply:
Yes, thats it. And when you state it that way, maybe the interface should be tweaked a bit. Open to ideas.

Reply:
One option might be to make the `getRuntimeFields` method public and call that directly, since it seems like that's what we care about here. Or even just passing `runtime: true` explicitly here might make it a bit clearer even though it's already the default.

Reply:
Made getRuntimeFields public